.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release Docker
+on:
+  workflow_run:
+    workflows: ["Renderscript"]
+    branches: [master]
+    types:
+      - completed
+
+env:
+  COMMIT_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: Release
+    needs: tests
```

Comment:
Does this work with a job outside of this workflow ?

Reply:
dry-run seems to work (at least trigger)

Reply:
Cool nice to know !